packages/gatsby/src/commands/serve.ts

```diff
@@ -132,7 +132,7 @@ module.exports = async (program: IServeProgram): Promise<void> => {
   if (functions) {
     app.use(
       `/api/*`,
-      multer().none(),
+      multer().any(),
```

Comment:
(at some later date, if these are basically the same behavior -- probably worth extracting and re-using)

Reply:
also on cloud 👍  https://app.clubhouse.io/gatsbyjs/story/29550/create-a-gatsby-functions-runtime-package-to-share-between-dev-env-gatsby-cloud-and-other-implementors?vc_group_by=day&ct_workflow=all